click/termui.py

```diff
@@ -390,10 +390,15 @@ def clear():
     if not isatty(sys.stdout):
         return
     # If we're on Windows and we don't have colorama available, then we
-    # clear the screen by shelling out.  Otherwise we can use an escape
+    # clear the screen by invoking `cls`.  Otherwise we can use an escape
     # sequence.
     if WIN:
-        os.system('cls')
+        import subprocess
+        try:
+            subprocess.call(['cls'])
```

Comment:
Can use `check_call` here, it raises a more specific error.

Reply:
Done.